evanxd.md

```diff
@@ -1,22 +1,29 @@
-## 08/08 ~ 08/12 ##
+## 08/15 ~ 08/19 ##
+
+* [Fennec]
+  - Bug 1280184 - Android: uninstalling firefox will delete all downloaded files, very annoying
+    - Still don't know why files are removed after uninstall the app. We use `Environment.getExternalStoragePublicDirectory` to store our downloaded files. Maybe we have code somewhere to delete them?
+  - Bug 1295018 - Android's Keyboard shouldn't learn new words when Private Tab is opened
+    - The issue might not to be resolved because "Use autocomplete but do not learn from this input" might be not possible.
 
 * [Firefox]
   - Dev Tools
-    - Bug 1284838 - Reps: render events more uniformly
+    - Bug 1289062 - Order the Event's properties
       - review+
+    - Bug 1295491 - Remove Reference() component in grip-array.js
+      - landed
+    - Bug 1284838 - Reps: render events more uniformly
+      - landed
     - Bug 1282465 - Reps: fix or remove recursive handling in ArrayRep and Obj rep
-      - review+
-    - Bug 1289062 - Order the Event's properties
-      - feedback+
+      - landed
   - Location Bar
     - Bug 1256074 - Always present a 'search' option, even location bar's contents are detected as URLs
-      - Looking for the code determine location bar treat the string as an URL or a keyword.
+      - After investigated, we could add two `richlistitem` elements(moz-action:visiturl and moz-action:searchengine ones) on the `#PopupAutoCompleteRichResult` panel when the `moz-action` is `visiturl`.
```

Comment:
👍 

Instead of waiting for a needinfo, how about let's just submit a patch and set ui-review?